### DISCUSSION TITLE
Find the odd one

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#585] [Galaxy Trivia] [answered]
**NGAAORKO**
**AOLKA**
**STUPPLAY**
**SHITROC**

**Comment by Veercodeprog** [comment]
STUPPLAY

**Reply by ParanoidUser** [reply]
Any particular reason for choosing this option?

**Reply by Veercodeprog** [reply]
Cause it doesn't form any word,others can be turned into meaningful words.

**Comment by thuongtruong1009** [comment, marked as answer]
The odd one out is "STUPPLAY".

All the other options can be rearranged to form names of countries:

"NGAAORKO" can be rearranged to spell "KOROGANIA" (a fictional country).
"AOLKA" can be rearranged to spell "KOALA" (an animal found in Australia).
"SHITROC" can be rearranged to spell "CROATIA" (a country in Europe).
However, "STUPPLAY" cannot be rearranged to form the name of any country.

**Comment by peterbaccaro** [comment]
1) **KANGAROO**
2) **KOALA**
3) **PLATYPUS**
4) **OSTRICH**

The OSTRICH is the odd one among KANGAROO, KOALA, PLATYPUS, and OSTRICH.

The KANGAROO, KOALA, and PLATYPUS are all native to Australia and are marsupials, meaning they give birth to relatively undeveloped young that continue to develop outside the womb in a pouch. The OSTRICH, on the other hand, is a flightless bird that is native to Africa and does not have a pouch or give birth to undeveloped young.

Additionally, the OSTRICH is the only bird in the group, while the KANGAROO, KOALA, and PLATYPUS are all mammals. The OSTRICH is also the largest bird and has distinct features such as its long neck, long legs, and inability to fly, which sets it apart from the other three animals.

**Reply by ParanoidUser** [reply]
Bravo! 👏👏
